test/parallel/test-buffer-bigint64.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+const buf = Buffer.allocUnsafe(8);
+
+['LE', 'BE'].forEach(function(endianness) {
+  // should allow simple BigInts to be written and read
+  let val = 123456789n;
+  buf['writeBigInt64' + endianness](val, 0);
+  let rtn = buf['readBigInt64' + endianness](0);
+  assert.strictEqual(val, rtn);
+
+  // should allow INT64_MAX to be written and read
+  val = 9223372036854775807n;
```

Comment:
Arguably a little more Obviously Correct when written in hex or as `2n ** 63n - 1n`.